src/NuGet.Services.EndToEnd/AutoCompleteResultTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using NuGet.Services.EndToEnd.Support;
+using NuGet.Versioning;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NuGet.Services.EndToEnd
+{
+    public class AutoCompleteResultTests
+        : IClassFixture<TrustedHttpsCertificatesFixture>
+    {
+        private readonly TestSettings _testSettings;
+        private readonly ITestOutputHelper _logger;
+        private readonly Clients _clients;
+
+        public AutoCompleteResultTests(ITestOutputHelper logger)
+        {
+            _clients = Clients.Initialize();
+            _testSettings = TestSettings.Create();
+            _logger = logger;
+        }
+
+        [Theory]
+        [InlineData(true, null)]
+        [InlineData(false, null)]
+        [InlineData(true, "2.0.0")]
+        [InlineData(false, "2.0.0")]
+        public async Task V2PackageVersionsAutocompleteResultsMatchV3Results(
+            bool includePrerelease,
+            string semVerLevel)
+        {
+            // Arrange
+            const string id = "EntityFramework";
```

Comment:
End-to-end tests should not assume any specific packages existing that will never be pushed by the tests themselves.